state/rbac_endpoint_permission.go

```diff
@@ -60,6 +61,10 @@ func (k *RBACEndpointPermissionsCollection) Add(rbacEndpointPermission RBACEndpo
 		return errEndpointRequired
 	}
 
+	if utils.Empty(rbacEndpointPermission.Workspace) {
+		return errWorkspaceRequired
+	}
```

Comment:
Please remove this and instead add workspace as a required field in https://github.com/Kong/deck/blob/main/file/codegen/main.go.
While you are at it, could you also make RBAC role's name field to be always required?
Feel free to add stricter validation for fields that are always required for RBAC role and RBAC endpoint permission.